EIPS/eip-5289.md

```diff
@@ -0,0 +1,139 @@
+---
+eip: 5289
+title: Notary Interface
```

Comment:
```suggestion
title: Smart Notary Interface
```
After a second thought, I think we can add a **Smart** keyword before **Notary**: the benefit is that if the interface gets adoption, the wording can be easier to be used in area outside of blockchain (e.g. popular media) to refer to the fact this concept comes from smart contract, yet for such reader it also shows the relevance to the concept of notary, making it more spreadable and understandable. Just my two cents, feel free to reject.

Reply:
Why not `Ethereum Notary Interface` or something?  I don't think `Smart` really means much these days, I have a "Smart Light" which has nothing to do with Ethereum.

Reply:
```suggestion
title: Ethereum Notary Interface
```

Reply:
Ethereum notary interface it is!